doc/tutorial_ti_cc3200.md

```diff
@@ -500,7 +300,13 @@ You should be able to go back to your device page on Xively CPM and see that its
 
 ## What to do next?
 
-_More coming soon, for now [please visit our docs](http://developer.xively.com/docs) and view some other guides and please let us know any feedback or questions that you may have regarding this guide or Xively in general!_
+### Stage 2: How to use custom wolfSSL during connecting to Xively with CC3200
```

Comment:
How about not saying 'custom' wolfssl but a more secure configuration of WolfSSL with OCSP?